package/kernel/linux/modules/usb.mk

```diff
@@ -1375,6 +1376,30 @@ endef
 $(eval $(call KernelPackage,usbip-server))
 
 
+define KernelPackage/usb-chipidea
+  TITLE:=Host and device support for Chipidea controllers
+  DEPENDS:=+USB_GADGET_SUPPORT:kmod-usb-gadget
+  KCONFIG:=\
+	CONFIG_NOP_USB_XCEIV=y \
+	CONFIG_EXTCON \
+	CONFIG_USB_CHIPIDEA \
+	CONFIG_USB_CHIPIDEA_HOST=y \
+	CONFIG_USB_CHIPIDEA_UDC=y \
+	CONFIG_USB_CHIPIDEA_DEBUG=y \
+	CONFIG_USB_EHCI_HCD=n
+  FILES:=\
```

Comment:
this logic should not be represented inside the mk file. we can expect users to know what they are installing and not install both at the same time